google/api_core/client_options.py

```diff
@@ -106,6 +112,7 @@ def __init__(
         self.scopes = scopes
         self.api_key = api_key
         self.api_audience = api_audience
+        self.universe_domain = universe_domain
```

Comment:
should the logic here be:

self.universe_domain = universe_domain if universe_domain else googleapis.com

Reply:
I'd like to disagree because:
1 - If the universe domain in the client options is empty, then we want to check if it's provided via an Env variable.
2 - The default value is handled in the client code itself by introducing `GOOGLE_DEFAULT_UNIVERSE = "googleapis.com"`

Reply:
I see. Sounds good.